Algorithm/QCAlgorithm.History.cs

```diff
@@ -985,6 +986,24 @@ private IEnumerable<SubscriptionDataConfig> GetMatchingSubscriptions(Symbol symb
                 var entry = MarketHoursDatabase.GetEntry(symbol, new []{ type });
                 resolution = GetResolution(symbol, resolution);
 
+                if (!LeanData.IsCommonLeanDataType(type) && !type.IsAbstract && !subscriptions.Any(x => x.Symbol == symbol))
```

Comment:
> && !subscriptions.Any(x => x.Symbol == symbol)

Wonder why we would need this part? I think it's okay to expect the regression `HistoryAlgorithms` stats to change 👌